src/main/java/com/sda/zdjavapol107/statki/ShipsMain.java

```diff
@@ -0,0 +1,283 @@
+                                break;
+                            }
+                            default: {
+                                UserInterface.printSingleLine("Podano bledna wartosc");
+                                break;
+                            }
+                        }
+                    }
+                } while (endOfPlacesTheShips == false);
+            } while (endOfPlacesTheShips == false);
+            //TODO tworzenie nowego statku
+        }
+    }
+
+    private static void creatingNewUser() {
+        userName = scanner.next();
+        UserShipsGame userShipsGame = new UserShipsGame(userName);
+        //TODO tworzenie nowego rekordu typu User w bazie danych
+    }
+
+    private static void placesTheSingleShip(int sizeOfShip) {
+        UserInterface.selectedStartingPointOfCreatingShip();
+        userPointX = scanner.nextInt();
+        UserInterface.printSingleLine("Podaj numer kolumny (1 - 10): ");
+        userPointY = scanner.nextInt();
+        firstPointShip = new PointShipsGame(userPointX, userPointY);
+        do {
+            UserInterface.newShipDirection();
+            shipDirection = scanner.nextInt();
+            switch (shipDirection) {
+                case 1: {
+                    isTop = true;
+                    isRight = false;
+                    isDown = false;
+                    isLeft = false;
+                    endLoopCreatingShip = canShipBePlaces(sizeOfShip, shipDirection, firstPointShip);
+                    break;
+                }
+                case 2: {
+                    isTop = false;
+                    isRight = true;
+                    isDown = false;
+                    isLeft = false;
+                    endLoopCreatingShip = canShipBePlaces(sizeOfShip, shipDirection, firstPointShip);
+                    break;
+                }
+                case 3: {
+                    isTop = false;
+                    isRight = false;
+                    isDown = true;
+                    isLeft = false;
+                    endLoopCreatingShip = canShipBePlaces(sizeOfShip, shipDirection, firstPointShip);
+                    break;
+                }
+                case 4: {
+                    isTop = false;
+                    isRight = false;
+                    isDown = false;
+                    isLeft = true;
+                    endLoopCreatingShip = canShipBePlaces(sizeOfShip, shipDirection, firstPointShip);
+                    break;
+                }
+                default: {
+                    UserInterface.printSingleLine("Podano bledna wartosc");
+                    endLoopCreatingShip = false;
+                }
+            }
+        }while (!endLoopCreatingShip);
+    }
+
+    public static boolean canShipBePlaces(int shipSize /*1, 2, 3, 4*/, int shipDirection /* Gora, Prawo, Dol, Lewo */, PointShipsGame point){
+        //TODO sprawdzanie czy statek moze byc umiejscowiony w tym miejscu
```

Comment:
przez to ze ten kod nie jest podzielony na funkcje trudno sie domyslic co sie tutaj w ogole dzieje :D 
a jesli musicie robic komentarze w stylu ze ship direction to gora prawo itp, to ja proponuje zmaienic ship direcion na enuma :D 
nawet ship size mozecie zastapic enumem jesli wyda wam sie to czytelniejsze